lib/codegen.js

```diff
@@ -47,6 +47,25 @@ var getPathToMethodName = function(m, path){
     return m.toLowerCase() + result[0].toUpperCase() + result.substring(1);
 };
 
+var getSchemaDefinition = function(schema) {
+    var obj = {};
+
+    if (schema.type !== 'object' || !_.isObject(schema.properties)) {
+        return obj;
+    }
+
+    obj.properties = _(schema.properties).map(function(param, key){
```

Comment:
It looks like you don't need chaining here, it suffices with `_.map(schema.properties, ...)`.